samples/quickstart.js

```diff
@@ -19,27 +19,31 @@
 // Imports the Google Cloud client library
 const Bigtable = require('@google-cloud/bigtable');
 
-// Creates a client
-const bigtable = new Bigtable();
-
-// The name for the new instance
-const instanceName = 'my-new-instance';
-
-// Creates the new instance
-bigtable
-  .createInstance(instanceName, {
-    clusters: [
-      {
-        name: 'my-cluster',
-        location: 'us-central1-c',
-        nodes: 3,
-      },
-    ],
-  })
-  .then(() => {
-    console.log(`Instance ${instanceName} created.`);
-  })
-  .catch(err => {
-    console.log('ERROR:', err);
-  });
+// The name of the Cloud Bigtable instance
+const instanceName = 'my-bigtable-instance';
+// The name of the Cloud Bigtable table
+const tableName = 'my-table';
+
+(async () => {
+  try {
+    // Creates a Bigtable client
+    const bigtable = new Bigtable();
+
+    // Connect to an existing instance:my-bigtable-instance
+    const instance = bigtable.instance(instanceName);
+
+    // Connect to an existing table:my-table
+    const table = instance.table(tableName);
+
+    //Read a row from my-table using a row key
+    let [singleRow] = await table.row('r1').get();
+
+    // Print the row key and data (column value, labels, timestamp)
+    console.log(`Row key: ${singleRow.id}\nData: ${JSON.stringify(singleRow.data, null, 4)}`);
+
+  } catch (err) {
+    // Handle error performing the read operation
+    console.err(`Error reading row r1: ${err}`);
```

Comment:
For all the error logging, log the error as another parameter instead of using template strings. One stringifies the error, the other prints it with a stack trace

```js
> console.error(`some ${new Error('The Error')}`)
some Error: The Error
undefined
> console.error(`some`, new Error('The Error'))
some Error: The Error
    at repl:1:23
    at Script.runInThisContext (vm.js:91:20)
    at REPLServer.defaultEval (repl.js:317:29)
    at bound (domain.js:396:14)
    at REPLServer.runBound [as eval] (domain.js:409:12)
    at REPLServer.onLine (repl.js:615:10)
    at REPLServer.emit (events.js:187:15)
    at REPLServer.EventEmitter.emit (domain.js:442:20)
    at REPLServer.Interface._onLine (readline.js:285:10)
    at REPLServer.Interface._line (readline.js:633:8)
```